spyder/plugins/completion/kite/widgets/status.py

```diff
@@ -16,50 +16,52 @@
 from spyder.utils import icon_manager as ima
 from spyder.widgets.calltip import ToolTipWidget
 from spyder.widgets.status import StatusBarWidget
-from spyder.plugins.completion.kite.utils.status import NOT_INSTALLED
+from spyder.plugins.completion.kite.utils.status import (
+    check_if_kite_installed, NOT_INSTALLED)
 
 logger = logging.getLogger(__name__)
 
 
 class KiteStatusWidget(StatusBarWidget):
     """Status bar widget for Kite completions status."""
     BASE_TOOLTIP = _("Kite completions status")
-    DEFAULT_STATUS = _('not reacheable')
+    DEFAULT_STATUS = _('not reachable')
 
     def __init__(self, parent, statusbar, plugin):
         self.plugin = plugin
         self.tooltip = self.BASE_TOOLTIP
         super(KiteStatusWidget, self).__init__(parent, statusbar,
                                                icon=ima.get_kite_icon())
-        self.tooltipWidget = ToolTipWidget(self, as_tooltip=True)
-        self.set_value(None)
+        self.tooltip_widget = ToolTipWidget(self, as_tooltip=True)
+        is_installed, _ = check_if_kite_installed()
+        self.setVisible(is_installed)
 
     def set_value(self, value):
         """Return Kite completions state."""
-        kite_enabled = self.plugin.get_option('enable')
+        is_installing = self.plugin.is_installing()
+        installation_status = self.plugin.installation_cancelled_or_errored()
```

Comment:
Are you sure we should show the widget if the installation erred? That doesn't make much sense to me.

Could you explain your reasoning?